test/scripts/gen-token.mjs

```diff
@@ -46,19 +47,21 @@ function generateToken(identity) {
  * @param {string} accessToken The access token in JWT form.
  */
 function templateAccessToken(accessToken) {
-  const TEMPLATE = (token) => `export const token =\n  '${token}';`;
+  const TEMPLATE = (token) => `export const token =\n  '${token}';\n`;
   return TEMPLATE(accessToken);
 }
 
 /**
  * Main function. Executed on script start.
  */
 function main() {
-  const { identity } = parseScriptArgument();
+  const { identity, path } = parseScriptArgument();
   const accessToken = generateToken(identity);
   const accessTokenJwt = accessToken.toJwt();
   const templatedAccessTokenJwt = templateAccessToken(accessTokenJwt);
-  console.log(templatedAccessTokenJwt);
+  writeFileSync(path, templatedAccessTokenJwt, {
+    flag: 'wx' /** wx prevents overwriting existing files */,
```

Comment:
In what scenario are we going to write the token more than once in the same box?

Reply:
There shouldn't be a circumstance where this happens, but I was thinking if someone puts in the wrong path, let's say `test/TwilioVoiceReactNativeExample/src/index.tsx` or something, we wouldn't want to overwrite that file.